var/spack/repos/builtin/packages/lapackpp/package.py

```diff
@@ -6,6 +6,16 @@
 from spack import *
 import os
 
+# Each LAPACK++ version requires a specific BLAS++ version
+_versions = [
+    # LAPACK++,     BLAS++
+    ['master',     'master'],
+    ['2020.10.00', '2020.10.00'],
+    ['2020.10.01', '2020.10.00'],
```

Comment:
Is it true that LAPACK++ 2020.10.01 needs BLAS++ **2020.10.00** or should it be 2020.10.01 ? I don't know, just checking! Thanks!

Reply:
You are correct, that is an error.  lapack@2020.10.01 should use blaspp@2020.10.01 since that version is available.